src/routes/e2e/widget/modal/+page.svelte

```diff
@@ -3,6 +3,7 @@
   import { page } from '$app/stores';
 
   import Widget, { modal, journey, user } from '$package/modal';
+  import type { Response } from '$lib/widget/interfaces';
```

Comment:
Not sure I understand this change. Why are we importing `{ Response }` but then changing the usage in this file from `Response` to `response`?

Reply:
These are just bugs. My PR should address most of these anomalies in the test app routes.